### PR TITLE
Add order submodule

### PR DESCRIPTION
**Pull request by legendofa**

### CODE REVIEW

File: .gitmodules
Comment (delvh) [inline .gitmodules:1]: There's a `git clone <order repo>` missing

Comment (lesser1) [reply .gitmodules:1]: Where exactly? I cannot find any "git clone" commands in the repository anywhere... I'm confused.

Comment (delvh) [reply .gitmodules:1]: You are declaring a submodule but you didn't actually clone the repo.
Please run `git submodule add ../order` in the repo root folder

**Comment by delvh** [comment]
Is this PR dead?

**Comment by nk-coding** [comment]
> Is this PR dead?

hopefully not
it is still required imho
but tbh, opening a new one is probably easier than fixing the conflicts 😂

**Comment by nk-coding** [comment]
new one without conflicts

**Pull request closed without merging**